tools/src/icon4pytools/py2fgen/wrappers/dycore_wrapper.py

```diff
@@ -88,79 +69,79 @@ def profile_disable():
-    geofac_grg_x: gt4py_common.Field[[CellDim, C2E2CODim], gtx.float64],
-    geofac_grg_y: gt4py_common.Field[[CellDim, C2E2CODim], gtx.float64],
-    nudgecoeff_e: gt4py_common.Field[[EdgeDim], gtx.float64],
-    bdy_halo_c: gt4py_common.Field[[CellDim], bool],
-    mask_prog_halo_c: gt4py_common.Field[[CellDim], bool],
-    rayleigh_w: gt4py_common.Field[[KHalfDim], gtx.float64],
-    exner_exfac: gt4py_common.Field[[CellDim, KDim], gtx.float64],
-    exner_ref_mc: gt4py_common.Field[[CellDim, KDim], gtx.float64],
-    wgtfac_c: gt4py_common.Field[[CellDim, KHalfDim], gtx.float64],
-    wgtfacq_c: gt4py_common.Field[[CellDim, KDim], gtx.float64],
-    inv_ddqz_z_full: gt4py_common.Field[[CellDim, KDim], gtx.float64],
-    rho_ref_mc: gt4py_common.Field[[CellDim, KDim], gtx.float64],
-    theta_ref_mc: gt4py_common.Field[[CellDim, KDim], gtx.float64],
-    vwind_expl_wgt: gt4py_common.Field[[CellDim], gtx.float64],
-    d_exner_dz_ref_ic: gt4py_common.Field[[CellDim, KHalfDim], gtx.float64],
-    ddqz_z_half: gt4py_common.Field[[CellDim, KHalfDim], gtx.float64],
-    theta_ref_ic: gt4py_common.Field[[CellDim, KHalfDim], gtx.float64],
-    d2dexdz2_fac1_mc: gt4py_common.Field[[CellDim, KDim], gtx.float64],
-    d2dexdz2_fac2_mc: gt4py_common.Field[[CellDim, KDim], gtx.float64],
-    rho_ref_me: gt4py_common.Field[[EdgeDim, KDim], gtx.float64],
-    theta_ref_me: gt4py_common.Field[[EdgeDim, KDim], gtx.float64],
-    ddxn_z_full: gt4py_common.Field[[EdgeDim, KDim], gtx.float64],
-    zdiff_gradp: gt4py_common.Field[[EdgeDim, E2CDim, KDim], gtx.float64],
-    vertoffset_gradp: gt4py_common.Field[[EdgeDim, E2CDim, KDim], gtx.int32],
-    ipeidx_dsl: gt4py_common.Field[[EdgeDim, KDim], bool],
-    pg_exdist: gt4py_common.Field[[EdgeDim, KDim], gtx.float64],
-    ddqz_z_full_e: gt4py_common.Field[[EdgeDim, KDim], gtx.float64],
-    ddxt_z_full: gt4py_common.Field[[EdgeDim, KDim], gtx.float64],
-    wgtfac_e: gt4py_common.Field[[EdgeDim, KHalfDim], gtx.float64],
-    wgtfacq_e: gt4py_common.Field[[EdgeDim, KDim], gtx.float64],
-    vwind_impl_wgt: gt4py_common.Field[[CellDim], gtx.float64],
-    hmask_dd3d: gt4py_common.Field[[EdgeDim], gtx.float64],
-    scalfac_dd3d: gt4py_common.Field[[KDim], gtx.float64],
-    coeff1_dwdz: gt4py_common.Field[[CellDim, KDim], gtx.float64],
-    coeff2_dwdz: gt4py_common.Field[[CellDim, KDim], gtx.float64],
-    coeff_gradekin: gt4py_common.Field[[EdgeDim, E2CDim], gtx.float64],
-    c_owner_mask: gt4py_common.Field[[CellDim], bool],
-    cell_center_lat: gt4py_common.Field[[CellDim], gtx.float64],
-    cell_center_lon: gt4py_common.Field[[CellDim], gtx.float64],
-    edge_center_lat: gt4py_common.Field[[EdgeDim], gtx.float64],
-    edge_center_lon: gt4py_common.Field[[EdgeDim], gtx.float64],
-    primal_normal_x: gt4py_common.Field[[EdgeDim], gtx.float64],
-    primal_normal_y: gt4py_common.Field[[EdgeDim], gtx.float64],
+    vct_a: gtx.Field[gtx.Dims[dims.KHalfDim], gtx.float64],
```

Comment:
can we not use field_type_aliases here?

Reply:
I think Hannes is trying to reduce the dependencies of the tools/py2fgen of the model to an absolute minimum. So we should not introduce another one.